src/fosslight_scanner/fosslight_scanner.py

```diff
@@ -186,10 +190,14 @@ def run_scanner(src_path, dep_arguments, output_path, keep_raw_data=False,
     try:
         output_file_without_ext = os.path.join(final_excel_dir, output_file)
         final_report = f"{output_file_without_ext}{output_extension}"
-        if remove_src_data:
-            overwrite_excel(_output_dir, default_oss_name, "OSS Name")
-            overwrite_excel(_output_dir, url, "Download Location")
-        success, err_msg = merge_excels(_output_dir, final_report)
+        if output_extension == ".xlsx":
+            if remove_src_data:
+                overwrite_excel(_output_dir, default_oss_name, "OSS Name")
+                overwrite_excel(_output_dir, url, "Download Location")
```

Comment:
There is no `overwrite_excel `function in yaml, so the data is different when outputting the following commands to yaml and excel.
`fosslight all -o test_result_wget -w "https://github.com/LGE-OSS/example.git"`